packages/cli/test/scripts/update.test.js

```diff
@@ -320,14 +364,20 @@ describe('update script', function() {
       it('should upgrade multiple proxies and migrate them', async function() {
         // add non-migratable implementation for WithLibraryImpl contract
         await add({
-          contractsData: [{ name: 'UnmigratableImplV2', alias: 'WithLibraryImpl' }],
+          contracts: ['UnmigratableImplV2'],
           projectFile: this.projectFile,
         });
         await push({ network, txParams, networkFile: this.networkFile });
 
+        const UnmigratableImplV2 = this.networkFile.contract('UnmigratableImplV2').address;
```

Comment:
```suggestion
        const unmigratableImplV2 = this.networkFile.contract('UnmigratableImplV2').address;
```
I'd reserve upper-case for contract classes, and use lower case for instances (or addresses)